### PR TITLE
test(frontend): Cover practically all the Storybook tests with snapshots

### PR DESCRIPTION
**Pull request by Twixes**
## Changes

Wanted to enable snapshots just for the Saved Insights stories, since I'm gonna work on that. But since #13947 made snapshots more reliable, why not enable this for even more stories.

### CODE REVIEW

File: frontend/src/scenes/paths/Paths.tsx [hunk 1]
```diff
@@ -43,7 +43,9 @@ export function PathsComponent({ nodeCard }: PathsComponentProps): JSX.Element {
     useEffect(() => {
         setNodeCards([])
 
-        const elements = document?.getElementById(id)?.querySelectorAll(`.Paths svg`)
+        // Remove the existing SVG canvas(es). The .Paths__canvas selector is crucial, as we have to be sure
+        // we're only removing the Paths viz and not, for example, button icons.
```
Comment (Twixes) [inline frontend/src/scenes/paths/Paths.tsx:47]: Already discovered one weird bug thanks to these tests: turns out resizing the window or toggling the sidebar resulted in the User Paths node card kebab button (dropdown) being removed 🤔

File: frontend/src/lib/components/HedgehogBuddy/HedgehogBuddy.stories.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default {
     title: 'Components/Hedgehog Buddy',
     component: HedgehogBuddy,
     parameters: {
-        chromatic: { disableSnapshot: true },
+        testOptions: { skip: true }, // Hedgehogs aren't particularly snapshotable
```
Comment (thmsobrmlr) [inline frontend/src/lib/components/HedgehogBuddy/HedgehogBuddy.stories.tsx:8]: That's very offensive to hedgehogs

File: frontend/src/scenes/PreflightCheck/preflightLogic.tsx [hunk 3]
```diff
@@ -214,7 +214,10 @@ export const preflightLogic = kea<preflightLogicType>([
         siteUrlMisconfigured: [
             (s) => [s.preflight],
             (preflight): boolean => {
-                return Boolean(preflight && (!preflight.site_url || preflight.site_url != window.location.origin))
+                if (window.location.port === '6006') {
+                    return false // Disable this warning in Storybook
```
Comment (thmsobrmlr) [inline frontend/src/scenes/PreflightCheck/preflightLogic.tsx:218]: Not sure which warning that is, but this won't work for https://storybook.posthog.net/

File: frontend/src/scenes/paths/Paths.tsx (hunk 1 above)
Comment (thmsobrmlr) [reply frontend/src/scenes/paths/Paths.tsx:47]: Haha, good catch!

**Review by thmsobrmlr — APPROVED** [review]
Loving all these little improvements! Feels great to have all the stories tested, the storybook config in typescript, a paths bugfix, etc.

We should remove the duplicated tests (insights) from e2e-vrt (I can do that after this is in). scenes-app-insights--user-paths.png has been flaky for snapshots in the past, so let's see how this goes.

File: frontend/src/scenes/PreflightCheck/preflightLogic.tsx (hunk 3 above)
Comment (Twixes) [reply frontend/src/scenes/PreflightCheck/preflightLogic.tsx]: Good catch, updated this "fix" to work in that case too, and explained the reason

**Pull request merged**